src/Engine/Combat/CombatEngine.java

```diff
@@ -42,11 +63,12 @@ public static BattleSummary simulateBattleResults( Unit attacker, Unit defender,
    * @param isSim Determines whether to cap damage at the HP of the victim in question
    * @return A BattleSummary object containing all relevant details from this combat instance.
    */
-  public static BattleSummary calculateBattleResults(Unit attacker, Unit defender, GameMap map, int attackerX, int attackerY, boolean isSim)
+  public static BattleSummary calculateBattleResults(Unit attacker, Unit defender, GameMap map,
+                                                     boolean attackerMoved, int attackerX, int attackerY,
+                                                     boolean isSim)
   {
     int defenderX = defender.x; // This variable is technically not necessary, but provides consistent names with attackerX/Y.
     int defenderY = defender.y;
-    boolean attackerMoved = attacker.x != attackerX || attacker.y != attackerY;
```

Comment:
Why move this into the parameter list?

Reply:
I wanted to use path info to do it in `BattleLifecycle`, since that's a better indicator than whatever the unit's *current* position is (transports and launch and stuff).
I could apply the change more broadly, though that'd be a lot of fiddling with logic that doesn't really need to change yet AFAIK.

Reply:
> I wanted to use path info to do it in BattleLifecycle, since that's a better indicator than whatever the unit's current position is (transports and launch and stuff).

Isn't `Path` just an equal indicator of the unit's current position? The `BattleEvent` will always be created (and the movement status/battle results calculated) before the unit actually moves, and this check works for units at (-1, -1) or whatever. It feels like duplicate information.

Reply:
The `Path` will always have exactly correct position information (whereas the unit might be off-map)
You're right, realistically.
However, I have now realized the *real* reason: `calculateOneStrikeDamage()` needs it to be a param because Muriel does her combat assessments with fake units.

Reply:
Then `calculateOneStrikeDamage()` needs it. Why here?

Reply:
Ah, now I remember.
I was contemplated a more limited version of LAUNCH that would cause the unit to unload, act in place, and immediately re-load.
In that particular case, a 1-length Path would be the only way to know (before you actually *execute* the move) that the unit hasn't moved.

Reply:
Is that... something we want to allow? I don't think AW allows it, and you were just waxing eloquent about source accuracy.
At any rate, I'd prefer to add affordances for new capabilities along with those capabilities; otherwise, the needed hooks often end up different from what I anticipated anyway.

Reply:
> Is that... something we want to allow? I don't think AW allows it, and you were just waxing eloquent about source accuracy.

Not on LAUNCH, no. But I have ideas for things that would work quite nicely with that feature.
Imagine a transport that can cart a Battleship up onto the land so it can snipe from a better position :stuck_out_tongue: 

> At any rate, I'd prefer to add affordances for new capabilities along with those capabilities; otherwise, the needed hooks often end up different from what I anticipated anyway.

That's a fair take. I feel like I have a pretty good track record on that, but I also somewhat frequently contemplate stuff off and on for months before I actually code it lol

Reply:
> Imagine a transport that can cart a Battleship up onto the land so it can snipe from a better position

I prefer to save my nightmares for when I'm actually asleep, thank you very much.

Reply:
> I prefer to save my nightmares for when I'm actually asleep, thank you very much.

You're welcome?